ownCloudAppShared/AppLock/PasscodeSetupCoordinator.swift

```diff
@@ -40,6 +42,8 @@ public class PasscodeSetupCoordinator {
 	private var passcodeViewController: PasscodeViewController?
 	private var passcodeFromFirstStep: String?
 	private var completionHandler: PasscodeSetupCompletion?
+	private var minPasscodeDigits: Int = 4
```

Comment:
@hosy A bit confused by the fact, that number of required digits is freely configurable but min / max counts are hard-coded here.

Reply:
I think `minPasscodeDigits ` should not be changed, because it makes no sense to set less than 4 digits. But I agree I added a new setting to set the maximum number of possible digits (default is 6).
Thanks for your review here!